feedback/static/feedback.css

```diff
@@ -551,6 +558,145 @@ body.ios .feedback-message tr .glyphicon {
 	align-items: flex-end;
 }
 
+/* Popovers */
+.popover {
+	width: 45vw;
+	max-width: 600px;
+}
+@media (max-width: 700px) {
+	.popover {
+		width: 90vw;
+	}
+}
+.popover:has( .pos-right) {
+	left: auto !important;
+	right: 2rem !important;
+}
```

Comment:
Is it absolutely necessary to use `!important` here? It is not a good practice to use it in CSS, unless there is no other way to make the styles work properly. If there is no other (relatively easy) way to make this work, add a comment explaining why we use `!important` here. 

https://developer.mozilla.org/en-US/docs/Web/CSS/important#best_practices

Reply:
Good point. The default location is dynamically defined inline, so for the conversation preview it needs to be defined with important, but I'm able to create workarounds for the other popovers.